RedHat-UX/red-hat-design-system
# docs: added includes for a11y pages

…each element

## What I did

Added the following files to `docs/_includes/accessibility` so that the accessibility tab can use them:
- 2.1.1-A.md
- 2.1.3-AAA.md
- 2.4.3-A.md
- 2.5.5-AAA.md
- ariaguide.md
- wcag.md


## Testing Instructions

<strike>Verify that the content in the [Badge accessibility tab](https://deploy-preview-970--red-hat-design-system.netlify.app/elements/badge/accessibility/) includes the following: </strike>

<hr>

A logical focus order helps keyboard users operate our websites. Elements need to receive focus in an order that preserves meaning, therefore the focus order should make sense and not jump around randomly. Interactive elements within expanded sections are automatically added to the focus order.

## ARIA Authoring Practices Guide (APG)
Learn to use the accessibility semantics defined by the [Accessible Rich Internet Application]([https://www.w3.org/WAI/fundamentals/](https://www.google.com/url?q=https://www.w3.org/WAI/fundamentals/&sa=D&source=docs&ust=1683060916306027&usg=AOvVaw1FHKJt8_gVS93hsJTvAPxj)) (ARIA) specification to create accessible web experiences.

## Web Content Accessibility Guidelines
<em>Understanding documents</em> provide detailed explanations for [Web Content Accessibility Guidelines](https://www.google.com/url?q=https://www.w3.org/WAI/WCAG21/Understanding/&sa=D&source=docs&ust=1683048326906694&usg=AOvVaw3mISGyx2SQ_TVY_MjttAhz) (WCAG) guidelines and success criteria.
- [Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard) (Level A, 2.1.1)
- [Keyboard (No Exception)](https://www.w3.org/WAI/WCAG21/Understanding/keyboard-no-exception) (Level AAA, 2.1.3)
- [Focus Order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order) (Level A, 2.4.3)
- [Target Size](https://www.w3.org/WAI/WCAG21/Understanding/target-size) (Level AAA, 2.5.5)
<hr>

## Notes to Reviewers
- <strike>I temporarily added a [`30-accessibility.md`](https://github.com/RedHat-UX/red-hat-design-system/pull/970/files#diff-cd313364ed4db36b9ef2c9d9fbbff222ba7d29536ce844f2c4b62370f6678eb9) page to `rh-badge` just so that we can check these files. Once it looks good I'll delete that file.</strike>
- We can just add other WCAG links as we need them later, but those 4 seem to be applicable in the most cases.
